packages/core/src/Stores/client-store.js

```diff
@@ -1172,7 +1179,7 @@ export default class ClientStore extends BaseStore {
                 );
             can_upgrade_to = canUpgrade('svg', 'iom', 'malta', 'maltainvest');
             if (can_upgrade_to) {
-                type = can_upgrade_to === 'maltainvest' ? 'financial' : 'real';
+                type = can_upgrade_to === 'maltainvest' ? MARKET_TYPES.FINANCIAL : 'real';
```

Comment:
Kindly change the hard coded `maltainvest` as well.

Reply:
The unused content and code will be removed in a separate PR as we're planning to keep things separate and unsupported landing companies like iom & malta will be removed. Then we'll create a config as well.